docs/model_developers/modules.rst

```diff
@@ -329,3 +329,41 @@ for supplying the agents with the appropriate ``inmanta_plugins`` packages.
 The only exception to this rule is when using the ``inmanta export`` command. It exports a project and all its modules'
 ``inmanta_plugins`` packages to the orchestrator server. When this method is used, the orchestrator does not install any modules
 from the Python package repository but instead contains all Python code as present in the local Python environment.
+
+Configure a project to install modules form a private python package repository
+--------------------------------------------------------------------------------
+
+Modules v2 are Python packages that can be installed from private Python package repositories, which may require authentication. This section explains the steps to install such modules.
+
+Create a file named ``.netrc`` in the orchestrator's file system, specifically in the "inmanta" directory. 
```

Comment:
We need to be more specific about the directory where this file needs to be created. There are many "inmanta" directories on the orchestrator.
```suggestion
Create a file named ``/var/lib/inmanta/.netrc`` in the orchestrator's file system. 
```